tests/common/mod.rs

```diff
@@ -0,0 +1,25 @@
+use play_reading_party::get_server;
+use sqlx::{Pool, Postgres};
+use std::net::TcpListener;
+
+// Create a test application on a random port using the given database pool
+pub async fn spawn_app(pool: &Pool<Postgres>, end_url: &str) -> TestApp {
```

Comment:
nit: would renaming `spawn_app` to something like `setup_test_environment` make more sense now that this is doing more than just spawning the server?